lib/xandra/connection.ex

```diff
@@ -267,8 +337,9 @@ defmodule Xandra.Connection do
     if warnings != [] do
       metadata =
         state
-        |> Map.take([:address, :port, :current_keyspace])
+        |> Map.take([:port, :current_keyspace])
         |> Map.put(:query, query)
+        |> Map.put(:host, state.address)
```

Comment:
I don't think we should rename `:address` to `:host`. I know it's what is written in https://github.com/lexhide/xandra/issues/283, but I think it was just an oversight. It's sort of a breaking change, but I don't really see a good reason to do it 🤔